frontend/javascripts/oxalis/controller.js

```diff
@@ -120,9 +124,16 @@ class Controller extends React.PureComponent<PropsWithRouter, State> {
         ) {
           Toast.error(messages["tracing.compound_project_not_found"], { sticky: true });
         }
-        // Don't throw errors for errors already handled by the model
-        // or "Not Found" errors because they are already handled elsewhere.
+        if (
+          error.organizationToSwitchTo != null &&
+          this.props.user != null &&
+          this.props.initialCommandType.owningOrganization !== this.props.user.organization
```

Comment:
Isn't `this.props.initialCommandType` a `TraceOrViewCommand` which doesn't have `owningOrganization` for the `TRACE` case? I would have thought, that flow complains about this or am I missing something? 

```suggestion
          error.organizationToSwitchTo !== this.props.user.organization
```

would make more sense, I think?

Reply:
Oh, thanks for catching this "problem". In the previous version where the API route did not exist I, this check was simple to avoid suggesting the user to the organization he was already in. In the `TRACE` case, the `owningOrganization` prop would simply be null and therefore the check would have still succeeded. At least this was the plan.

In the current version, your suggestion should be correct.

However, I think we can remove the whole check of `error.organizationToSwitchTo !== this.props.user.organization` in the current version, as we can assume, 
1. that the backend does not suggest to switch to the already active organization
2. The whole call to ask for the organization to switch to would only be executed once the loading fails. Thus if the user was already in the correct organization, the user should be able to see the dataset, if the user is allowed to.

Reply:
I added a line to the backend that ensures it never suggests to switch to the orga the user is already in :)

Reply:
> that the backend does not suggest to switch to the already active organization

@fm3 is this the case? at least, I cannot read the behavior in the back-end code right now. if the back-end doesn't do this, the frontend can easily do this here in my opinion.
If this is not handled, some other error could send the user into an endless loop of "loading fails" -> "try this organization" -> "loading fails" -> ...

Reply:
ah, race condition during writing comments :see_no_evil: awesome! :+1: